AdityaFlashCards/AdityaFlashCards/Application.cs

```diff
@@ -0,0 +1,281 @@
+        Db = new DatabaseManager(configurationBuilder);
+    }
+
+    internal void MainMenu()
+    {
+        bool runApp = true;
+        while (runApp)
+        {
+            Console.Clear();
+            string option = Display.GetSelection("Hi! What do you wish to do?", new List<string> { "Manage Stacks", "Manage Flashcards", "Study", "View Study Sessions", "Quit" });
+            switch (option)
+            {
+                case "Manage Stacks":
+                    ManageStacks();
+                    break;
+                case "Manage Flashcards":
+                    ManageFlashCards();
+                    break;
+                case "Study":
+                    HandleStudy();
+                    break;
+                case "View Study Sessions":
+                    HandleViewStudySession();
+                    break;
+                case "Quit":
+                    Console.WriteLine("Goodbye");
+                    Environment.Exit(0);
+                    break;
+                default:
+                    break;
+            }
+        }
+    }
+
+    internal void ManageStacks()
+    {
+        string selection = Display.GetSelection("Manage Stacks", new List<string> { "View Stacks", "Create New Stacks", "Delete Stack", "Return to Main Menu" });
+        switch (selection)
+        {
+            case "View Stacks":
+                List<Stack> allStacks = Db.GetAllStacks();
+                Console.WriteLine("----------Following is the list of all stacks ---------------");
+                List<string> list = new List<string>();
+                foreach (var stack in allStacks)
+                {
+                    list.Add(stack.Name.ToString());
+                }
+                list.Add("Go to Main Menu");
+                string stackName = Display.GetSelection("Select a stack to view from the list", list);
+                Console.Clear();
+                if (stackName != "Go to Main Menu")
+                {
+                    Console.WriteLine("----------Following are the flashcards in your stack ---------------");
+                    Console.WriteLine("----------StackName: {0}---------- ", stackName);
+                    List <FlashCardDtoStackView> Flashcards = Db.GetFlashCardsForGivenStack(stackName);
+                    if (Flashcards.Count > 0)
+                    {
+                        Display.DisplayFlashCards(new string[] { "PositionInStack", "Front Side", "Back Side" }, Flashcards);
+                        Console.Write("Press any key to continue...");
+                        Console.ReadLine();
+                    }
+                    else
+                    {
+                        Console.Write("The stack you wanted to see is empty.Press any key to continue...");
+                        Console.ReadLine();
+                    }
+                }
+                break;
```

Comment:
There's quite a lot of code in the case statements. You could improve readability by extracting into methods.